python/api/utils.py

```diff
@@ -37,6 +37,13 @@
 )
 from cm.upgrade import get_upgrade
 
+REASON_MSG_DICT = {
```

Comment:
Should we really define some list of possible reasons ? May be we can just use some arbitrary string, save it in DB, and pass it for API request